.github/workflows/elixir.yml

```diff
@@ -2,56 +2,59 @@ name: Elixir CI
 
 on:
   push:
-  pull_request:
     branches:
       - main
-
-env:
-  ELIXIR_LATEST: '1.16.0'
+  pull_request:
 
 permissions:
   contents: read
 
 jobs:
   build:
-    name: Build and test with elixir ${{ matrix.elixir }} otp ${{ matrix.otp }}
```

Comment:
why remove this?

Reply:
The name is too long and truncated as we can't see the OTP version and `lint` flag.

From:

![image](https://github.com/user-attachments/assets/07c59cf4-3acf-4730-bb08-a02f50ffab7e)

To:

![image](https://github.com/user-attachments/assets/715fc848-c9ee-4701-a9d8-91c52eef6ab3)